ravenframework/Optimizers/acquisitionFunctions/AcquisitionFunction.py

```diff
@@ -11,13 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""
-"""
```

Comment:
These will need to be filled out with a comment describing a file, rather than removed.
Ditto for the other ones mentioned by pylint:
```
Framework
************* Module ravenframework.Optimizers.acquisitionFunctions.AcquisitionFunction
ravenframework/Optimizers/acquisitionFunctions/AcquisitionFunction.py:1:0: C0114: Missing module docstring (missing-module-docstring)
************* Module ravenframework.Optimizers.acquisitionFunctions.ExpectedImprovement
ravenframework/Optimizers/acquisitionFunctions/ExpectedImprovement.py:1:0: C0114: Missing module docstring (missing-module-docstring)
************* Module ravenframework.Optimizers.acquisitionFunctions.Factory
ravenframework/Optimizers/acquisitionFunctions/Factory.py:1:0: C0114: Missing module docstring (missing-module-docstring)
************* Module ravenframework.Optimizers.acquisitionFunctions.LowerConfidenceBound
ravenframework/Optimizers/acquisitionFunctions/LowerConfidenceBound.py:1:0: C0114: Missing module docstring (missing-module-docstring)
************* Module ravenframework.Optimizers.acquisitionFunctions.ProbabilityOfImprovement
ravenframework/Optimizers/acquisitionFunctions/ProbabilityOfImprovement.py:1:0: C0114: Missing module docstring (missing-module-docstring)
```

Reply:
I believe I have addressed this, but will wait until it is no longer an issue to mark this as resolved.

Reply:
It is fixed, thank you.